rcl_action/src/rcl_action/action_server.c

```diff
@@ -645,8 +650,14 @@ rcl_action_expire_goals(
       ret_final = RCL_RET_ERROR;
       continue;
```

Comment:
640-651 can not be deleted ?

Reply:
Do you means lines related to ` goal_time = _goal_info_stamp_to_nanosec(info_ptr);` ?

Reply:
the check if the goal is active, and the code around the info_ptr should now be not needed any more.

Reply:
Yes. These code can be deleted. 
I will update code. Thanks